scripts/deploy-production.js

```diff
@@ -14,113 +22,89 @@ const main = async () => {
     return;
   }
 
-  // Print all commits between the last tag and the current commit
   const commitMessages = await getCommitMessages();
-  let shouldDeployLocal = false;
+
+  // If there are no commit messages, ask the user if they want to delete and re-push the latest tag, thereby forcing a deploy with no new commits.
   if (commitMessages.length === 0) {
-    // If there are no commit messages, ask the user if they want to deploy anyway
     const response = await prompts({
       type: "confirm",
       name: "value",
-      message: "No new commits. Do you want to deploy anyway?",
+      message:
+        "No new commits. Do you want to deploy the most recent existing tag?",
```

Comment:
Out of curiosity what happens if we push a duplicate tag to evergreen? @ablack12